### DISCUSSION TITLE
Filters

### DISCUSSION THREAD

**Discussion by rubenslkirchner** [macgile/DataGridFilter#135] [General]
Hi all!
I'm attempting to have the FilterDataGrid display the 'ThrottleNotchPosition' property of the sensors within the event, but unfortunately, the filter is not showing any values and remains 'blank'. The cell value is displaying the correct throttle notch value (with  converter) but the filter still not working. Any idea what is wrong?

<FilterDataGrid AutoGenerateColumns="False" ItemsSource="{Binding Events}">
    <DataGrid.Columns>
        

       <filterDataGrid:DataGridTemplateColumn
                        Header="{x:Static t:Translation.ThrottleNotch}"
                        IsColumnFiltered="True"
                        FieldName="ThrottleNotchPosition">
                        <filterDataGrid:DataGridTemplateColumn.CellTemplate>
                            <DataTemplate DataType="{x:Type domains:SensorsEvent}">
                                <TextBlock Text="{Binding Sensors.ThrottleNotchPosition, Converter={StaticResource genericenumtextconverter}}"/>
                            </DataTemplate>
                        </filterDataGrid:DataGridTemplateColumn.CellTemplate>
                    </filterDataGrid:DataGridTemplateColumn>
    </DataGrid.Columns>
</FilterDataGrid>

![image](https://github.com/macgile/DataGridFilter/assets/7251969/8bf7ee03-9acb-48c5-9283-7bfb540b1d53)

**Comment by macgile** [comment]
hi,

What kind of values are these?
Can you give me a sample of the ThrottleNotchPosition values?

**Reply by rubenslkirchner** [reply]
Hi @macgile 

ThrottleNotchPostion is an enum 

enum ThrottleNotchPosition {
Undefined = 0,
Idle = 1,
P1 = 2,
P2 = 3,
...
}

I have other columns that are classes with string properties that will have the same issue.

**Reply by rubenslkirchner** [reply]
@macgile any ideia?

**Comment by macgile** [comment]
hi,

Filter Datagrid does not currently support combobox columns
This is what you're trying to do

**Comment by iamemon123** [comment]
I have installed FilterDataGrid 1.2.7 from Nuget (.Net Framework 4.8. ).

I am facing this issue 

System.Windows.Markup.XamlParseException: Could not load file or assembly 'FilterDataGrid, PublicKeyToken=42060d65a6cc17c2' or one of its dependencies. The system cannot find the file specified. ---> System.IO.FileNotFoundException: Could not load file or assembly 'FilterDataGrid, PublicKeyToken=42060d65a6cc17c2' or one of its dependencies. The system cannot find the file specified.

what should be the cause.
